cloudformation/apps/instance-profiles.yaml

```diff
@@ -10,6 +10,18 @@ Parameters:
       Type: String
```

Comment:
Move the stack-manager specific instance profiles to `cloudformation/apps/stack-manager/instance-profiles.yaml` . Ditto for playbook.